doc/level1.rst

```diff
@@ -10,8 +10,8 @@ mails while supporting users who want to explicitly turn on
 encryption.
 
 For ease of implementation and deployment, Level 1 does not support
-multi-device configurations.  We intend to :doc:`support the multi-device
-use case (and other features) as part of Level 1<next-steps>`.  We
+multi-device synchronisation.  We intend to :doc:`support the multi-device
```

Comment:
i am merging this but think that we could say that Level 1 only supports manual multi-device synchronisation where a user can transfer a secret from one device to another ...